product_listings_manager/products.py

```diff
@@ -324,6 +334,10 @@ def get_product_listings(db, product_label, build_info):
 
         # debuginfo only
         rpms_debug = [rpm for rpm in rpms if koji.is_debuginfo(rpm["name"])]
+        if variant in VARIANTS_WITHOUT_FAST_SLOW_DEBUGINFO:
```

Comment:
Sure, but it makes more sense to me to apply this to all fast/slowdebug packages. Why do this only for specific packages?

Reply:
I guess this is OpenJDK specific thing anyway. Are there any other fast/slowdebug packages?

Reply:
As this "-fastdebug-debuginfo", "-slowdebug-debuginfo" is specific for the RPM, it's better to mention in logic, otherwise we will easy losing track after a long time.

Actually, I don't prefer this hardcode change in PLM, as I saw other cases, we need to think about a good solution to forbid the problem happen again and again. Like monitor the ET umb message, match the packages and check debuginfo correctness and call script to fix.

Reply:
This can not be monitored as the debug kind rpm's names don't have debug or fast|slow text in the NVR, so when the build is added to ET, if we want to know about if there are any slow|fast debug packages, we need to scan on the brew files level.
So the best place to solve this problem might be on PLM itself.